README.md

```diff
@@ -47,30 +47,55 @@ The fields in the table below can be used in these parts of STAC documents:
-[Best Practices section](#best-practices) so that, for example, GDAL can read your data without issues.
-
-Also, [version 1.0.0](https://github.com/stac-extensions/projection/releases/tag/v1.0.0) of this extension
-required `proj:epsg` (either as integer or null) in Item Properties.
-This is not required anymore, but it is still recommended to additionally provide an EPSG code whenever available.
+[Best Practices section](#best-practices) so that, for example, GDAL can read your data without issues. 
 
 Generally, it is preferrable to provide the projection information on the Asset level
 as they are specific to assets and may not apply to all assets.
 For example, if you provide a smaller and unlocated thumbnail, having the projection information in the Item Properties
 would imply that the projection information also applies to the thumbnail if not specified otherwise in the asset.
-You may want to add the EPSG code to the Item Properties though as this would provide an easy way to
-filter for specific EPSG codes in an API.
-In this case you could override the EPSG code for the thumbnail on the asset level.
+You may want to add the `proj:code` to the Item Properties though as this would provide an easy way to
+filter for specific projection codes in an API. In this case you could override the `proj:code` for the thumbnail on the asset level.
 
 ### Additional Field Information
 
 #### proj:epsg
 
-A Coordinate Reference System (CRS) is the data reference system (sometimes called a
-'projection') used by the asset data, and can usually be referenced using an 
-[EPSG code](https://en.wikipedia.org/wiki/EPSG_Geodetic_Parameter_Dataset).
-A great tool to help find EPSG codes is [epsg.io](http://epsg.io/).
+**Notice**: This field has been removed in V2.0.0. See [proj:epsg Migration to V2](#projepsg-migration-to-projcode).
 
-This field SHOULD be set to `null` in the following cases:
+#### proj:epsg migration to proj:code
+`proj:epsg` is removed in version 2.0.0 of this extension. Please use `proj:code`. For example, the following:
+
+```json
+{
+  ...
+  "proj:epsg": 32659,
+  ...
+}
+```
+would be represented as:
+```json
+{
+  ...
+  "proj:code": "EPSG:32659",
+  ...
+}
+```
+
+#### proj:code
+
+Projection codes are identified by a string. The [proj](https://proj.org/) library defines projections 
+using "authority:code", e.g., "EPSG:4326" or "IAU_2015:30100". Different projection authorities may define 
+different string formats. Examples of known projection authorities, where when can find well known codes that 
+clients are likely to support are listed in the following table.
+
+| Authority Name                          | URL                                                        |
+| --------------------------------------- | ---------------------------------------------------------- |
+| European Petroleum Survey Groups (EPSG) | <http://www.opengis.net/def/crs/EPSG> or <http://epsg.org> |
```

Comment:
is the string in parens supposed to be the acronym, the authority value, or both? If the authority, it should be in a separate column also (though keeping the acronym is probably good too, since people know EPSG not the full name). Also, IAU here instead of IAU_2015 like other places

Reply:
I like the separate column for the part before the double colon.

Reply:
So, IAU_2015 is the authority because IAU is versioned temporally in a way that other authorities do not. I included the parenthetical initialism to help users who may be more familiar with the initialism. I'll remove the `/2015` from the IAU URL as that could cause confusion.

I would not try to track, in this extension, the left and right side of the shortcode column. That should be coming from, for the stack that has access to the proj library, proj. For tooling that does not have access to the proj library, these short codes really don't matter and the wkt/projjson strings are going to get parsed.